src/js/signin/containers/Main.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import environment from '../../common/helpers/environment.js';
+import { getUserData, addEvent, handleLogin, getLoginUrls } from '../../common/helpers/login-helpers';
+
+import { updateLoggedInStatus, updateLogInUrl, updateVerifyUrl, updateLogoutUrl, updateLogInUrls } from '../../login/actions';
+import Signin from '../components/Signin';
+import Verify from '../components/Verify';
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.setMyToken = this.setMyToken.bind(this);
```

Comment:
On Rainbows we've mostly switched to writing class methods like this, to avoid these bind calls:

```js
setMyToken = (event) => {
...
}
```

Reply:
I'd like to see that too. Sounds like a potential future sweep that can be done across all products. Might be good to bring up at the next FE dev meeting.

Reply:
Agree. These issues are related to code style but more of a thing we could establish across the entire team instead of having variance in different apps